modules/tinymce/src/core/main/ts/events/EventUtils.ts

```diff
@@ -69,6 +70,12 @@ const clone = <T extends PartialEvent>(originalEvent: T, data?: T): T => {
     event.composedPath = () => originalEvent.composedPath!();
   }
 
+  // The composed path can't be cloned, so delegate instead
```

Comment:
Please be careful when copy/pasting code, this comment is now incorrect 😛 

- [ ] change comment